docs/Project.toml

```diff
@@ -1,6 +1,7 @@
 [deps]
 BSplineKit = "093aae92-e908-43d7-9660-e50ee39d5a0a"
 CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
```

Comment:
This shouldn't be needed either, ForwardDiff is only needed in the main project.